app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (OutdatedVaccineError,
+                        NotVaccinatedError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitors: dict) -> str:
+        if "vaccine" not in visitors:
+            raise NotVaccinatedError("NotVaccinatedError")
+        if visitors["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        if not visitors["wearing_a_mask"]:
+            raise NotWearingMaskError("NotWearingMaskError")

```

Comment:
no need to repeat a name of exception in message, use more descriptive messages for exceptions

Reply:
![image](https://user-images.githubusercontent.com/103201010/219682293-a7d4cd9b-261b-4201-84e4-8d0514825ee1.png)